backend/app/controllers/api/v1/tutorings_controller.rb

```diff
@@ -165,7 +166,23 @@ def show
               end_time: a.end_time,
               is_booked: a.is_booked
             }
-          end
+          end,
+          enrolled_students: @tutoring.users.map do |user|
+            {
+              id: user.id,
+              photo_url: UserSerializer.new(user).serializable_hash[:data][:attributes][:profile_photo_url],
```

Comment:
```suggestion
              photo_url: user.profile_photo.attached? ? url_for(user.profile_photo) : nil,
              
```
No llamen a todo el serializer si solo quieren la url de la foto de perfil